Update bullet chart tutorial (#214)
patternfly/training-scenarios: react-charts/bullet-chart/intro.md

PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly.

> PatternFly provides a collection of React-based components you can use to build patterns with consistent markup, styling, and behavior.

Reviewer: ```suggestion
PatternFly provides a collection of React-based components that can be used to build patterns with consistent markup, styling, and behavior.
```
